NEWS.md

```diff
@@ -29,6 +29,7 @@
 - Typo.
 
 
+
```

Comment:
I wonder if we can control the number of empty lines added here.

Reply:
it was a mistake, it should be two lines, now fixed.